pkg/controller/service/controller_test.go

```diff
@@ -307,8 +307,6 @@ func TestSyncLoadBalancerIfNeeded(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.desc, func(t *testing.T) {
-			defer featuregatetesting.SetFeatureGateDuringTest(t, utilfeature.DefaultFeatureGate, serviceLoadBalancerFinalizerFeature, tc.enableFeatureGate)()
```

Comment:
Can we remove `serviceLoadBalancerFinalizerFeature` or add a TODO to remove in a follow-up PR?

Reply:
Thanks Andrew. Totally, removed `serviceLoadBalancerFinalizerFeature` as that is no longer needed.